cmd/oras/internal/display/status/progress/status.go

```diff
@@ -65,25 +65,25 @@ func newStatus() *status {
 	}
 }
 
-// NewStatusMessage generates a status for messaging.
-func NewStatusMessage(prompt string, descriptor ocispec.Descriptor, offset int64) *status {
+// newStatusMessage generates a status for messaging.
+func newStatusMessage(prompt string, descriptor ocispec.Descriptor, offset int64) *status {
 	return &status{
 		prompt:     prompt,
 		descriptor: descriptor,
 		offset:     offset,
 	}
 }
 
-// StartTiming starts timing.
-func StartTiming() *status {
+// startTiming starts timing.
+func startTiming() *status {
 	return &status{
 		offset:    -1,
 		startTime: time.Now(),
 	}
 }
 
-// EndTiming ends timing and set status to done.
-func EndTiming() *status {
+// endTiming ends timing and set status to done.
+func endTiming() *status {
 	return &status{
 		offset:  -1,
 		endTime: time.Now(),
```

Comment:
Other than making those private, I think we can get rid of those functional. I wrote those code and it sucks :)

Reply:
I mean move function content to `messenger.go`

Reply:
I'm going to move them to `messenger.go`. They don't do much, but I think the code reads better with them because of the select/case statement.

Reply:
Moved